apps/passport-server/src/util/devconnectTicket.ts

```diff
@@ -20,5 +20,9 @@ export function pretixTicketsDifferent(
     return true;
   }
 
+  if (oldTicket.is_consumed !== newTicket.is_consumed) {
+    return true;
+  }
```

Comment:
probably missing something obvious, but why aren't the other fields you added not included here?

Reply:
They don't vary independently of `is_consumed`. There's no harm in checking them, though, so I'll add that.